Draft-Accepted/feedback_completer_auto_discovery.md

```diff
@@ -0,0 +1,240 @@
+# Auto Discovery for Feedback provider and Tab-Completer
+
+## Motivation
+
+Today, to enable a feedback provider or a tab-completer for a native command,
+a user has to import a module or run a script manually, or do that in their profile.
+There is no way to auto-discover a feedback provider or a tab-completer for a specific native command.
```

Comment:
Should we consider predictors in this set of supported auto discoverable "things"?

Reply:
For example the completion predictor may be good to auto register since it doesnt really have any commands

Reply:
There won't be a specific trigger for any predictor, so if needed, a predictor module can just be placed under the `_startup_` folder of `Feedbacks` or `Completions`.

Think about it more, I guess we may want to unify the `_startup_` folders from `feedbacks` and `completions`, since they all will be loaded at startup. Maybe we should have a `startup` folder at the same level of `feedbacks` and `completions`, so any predictor/feedback provider/tab-completer that needs to be loaded at session startup can be put there.